newrelic.js

```diff
@@ -0,0 +1,55 @@
+var API_KEY = require('./.config.js');
+//console.log(API_KEY)
```

Comment:
You should probably kill most of the comments in this file.

Reply:
Got rid of the console log, but as I am still learning newrelic.js I will leave the additional comments in until final cleanup